### PR TITLE
rgw, qa: integrate object storage tests of Tempest with Ceph QA suite

### PR DESCRIPTION
**Pull request by rzarzynski**
This PR is an very early result of works on Tempest integration. We still need to:
- improve cleaning,
- load test list from file in order to facilitate test masking,
- add a trigger for `make check` or teuthology.

### CODE REVIEW

**Comment by yehudasa** [comment]
@mattbenjamin can you take a look at this one?

File: qa/workunits/rgw/tempest.sh [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  git clone --quiet -- ${TEMPEST_REPO_URL} ${TEMPEST_HOME_DIR}
+  cd ${TEMPEST_HOME_DIR} \
+      && git checkout ${TEMPEST_RELEASE}
+}
+
+tempest_make_venv()
+{
+  cd ${TEMPEST_HOME_DIR}
+
+  python tools/install_venv.py > /dev/null
```
Comment (rzarzynski) [inline qa/workunits/rgw/tempest.sh]: ```tools/install_venv.py``` has been recently removed from Keystone. ```tox``` should be used instead but it doesn't work for me. Following workaround may be employed:

```
virtualenv env
source env/bin/activate
pip install -e '.[ldap,memcache,mongodb]'
```

**Comment by mattbenjamin** [comment]
@rzarzynski sorry for the delay on this, is this still fully applicable?

**Comment by rzarzynski** [comment]
@mattbenjamin: it needs some small changes related to the Keystone deployment. However, what worries me a bit is the execution time that makes this unsuitable for the `make check` bot. Anyway, I think we could integrate with the Teuthology instead.

**Comment by liewegas** [comment]
@rzarzynski ping

**Comment by rzarzynski** [comment]
The Tempest integration with Teuthology has been merged in #16344. Closing this one.

**Pull request closed without merging**